packages/client/hmi-client/src/components/model/stockflow/tera-stockflow-tables.vue

```diff
@@ -0,0 +1,265 @@
+							data.payload?.mentions?.map((m) => m?.name).join(', ') ||
+							'--'
+						}}
+					</template>
+				</Column>
+				<Column header="Values">
+					<template #body="{ data }">
+						{{
+							data.payload?.values?.map((n) => n?.value?.amount).join(', ') ||
+							data.payload?.value_descriptions?.map((m) => m?.value?.amount).join(', ') ||
+							'--'
+						}}
+					</template>
+				</Column>
+				<Column header="Descriptions">
+					<template #body="{ data }">
+						{{
+							data.payload?.descriptions?.map((d) => d?.source).join(', ') ||
+							data.payload?.text_descriptions?.map((d) => d?.description).join(', ') ||
+							'--'
+						}}
+					</template>
+				</Column>
+				<Column field="payload.groundings" header="Concept">
+					<template #body="{ data }">
+						<template v-if="!data?.payload?.groundings || data?.payload?.groundings.length < 1"
+							>--</template
+						>
+						<template
+							v-else
+							v-for="grounding in data?.payload?.groundings"
+							:key="grounding.grounding_id"
+						>
+							{{ grounding.grounding_text }}
+							<a
+								target="_blank"
+								rel="noopener noreferrer"
+								:href="getCurieUrl(grounding.grounding_id)"
+								aria-label="Open Concept"
+							>
+								<i class="pi pi-external-link" />
+							</a>
+						</template>
+					</template>
+				</Column>
+			</DataTable>
+		</AccordionTab>
+		<AccordionTab>
+			<template #header>
+				Time
+				<span class="artifact-amount">({{ time.length }})</span>
+			</template>
+			<DataTable v-if="!isEmpty(time)" data-key="id" :value="time">
+				<Column field="id" header="Symbol" />
+				<Column field="units.expression" header="Unit" />
+			</DataTable>
+		</AccordionTab>
+	</Accordion>
+</template>
+
+<script setup lang="ts">
+import type { Initial, Model, ModelConfiguration, ModelParameter } from '@/types/Types';
+import { cloneDeep, groupBy, isEmpty } from 'lodash';
+import Accordion from 'primevue/accordion';
+import AccordionTab from 'primevue/accordiontab';
+import { computed, ref, watch } from 'vue';
+import { Dictionary } from 'vue-gtag';
+import { getCurieUrl } from '@/services/concept';
+import DataTable from 'primevue/datatable';
+import Column from 'primevue/column';
+import TeraParameterTable from '@/components/model/petrinet/tera-parameter-table.vue';
+import TeraInitialTable from '@/components/model/petrinet/tera-initial-table.vue';
+import { MiraModel, MiraTemplateParams } from '@/model-representation/mira/mira-common';
+import { emptyMiraModel } from '@/model-representation/mira/mira';
+import { getMMT } from '@/services/model';
+import Button from 'primevue/button';
+
+const props = defineProps<{
+	model: Model;
+	modelConfigurations?: ModelConfiguration[];
+	readonly?: boolean;
+}>();
+
+const emit = defineEmits(['update-model']);
+
+const mmt = ref<MiraModel>(emptyMiraModel());
+const mmtParams = ref<MiraTemplateParams>({});
+
+const transientModel = ref(cloneDeep(props.model));
```

Comment:
I think you are cloning this multiple times (e.g. there is one down by the watcher).